app/assets/stylesheets/refills/_flashes.scss

```diff
@@ -9,9 +9,10 @@ $success-color: #e6efc2 !default;
   color: darken($color, 60%);
   display: block;
   font-weight: 600;
-  margin-bottom: $base-spacing / 2;
+  margin-bottom: 0;
   padding: $base-spacing / 2;
   text-align: center;
+  clear: both;
```

Comment:
I would add `@include clearfix();` to the navigation rather than adding a clear to the item that follows the navigation.